build_openroad.sh

```diff
@@ -132,12 +133,12 @@ if [ "$build_method" == "DOCKER" ]; then
 
 # Local build
 elif [ "$build_method" == "LOCAL" ]; then
-  mkdir -p tools/build/yosys
-  (cd tools/yosys && $NICE make install -j$PROC PREFIX=../build/yosys CONFIG=gcc)
+  mkdir -p tools/install/yosys
+  (cd tools/yosys && $NICE make install -j$PROC PREFIX=../install/yosys CONFIG=gcc)
```

Comment:
could you make the PREFIX an absolute path? yosys-config just copies the PREFIX without making it an abspath

Reply:
What is yosys-config used for?

Reply:
yosys-config is used to build the yosys plugins, very handy, since it preserves the options used to build yosys. For example:
```
yosys-config --build oracle.so oracle.cc
```

Reply:
Makes sense. I'll change that.